### PR TITLE
Generating relaxed model to output directory

### PR DESCRIPTION
**Pull request by hcorg**

### CODE REVIEW

File: src/lib/reconstructor.cpp.orig [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+** GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License
+** along with this m_processogram.  If not, see <http://www.gnu.org/licenses/>.
+**
+****************************************************************************/
+
+#include "reconstructor.h"
```
Comment (psockoN7) [inline src/lib/reconstructor.cpp.orig]: This file is merge leftover.

File: src/lib/reconstructor.h.orig [hunk 2]
```diff
@@ -0,0 +1,71 @@
+/****************************************************************************
```
Comment (psockoN7) [inline src/lib/reconstructor.h.orig]: This file needs to go also, I believe.

File: src/lib/reconstructor.cpp.orig (hunk 1 above)
Comment (hcorg) [reply src/lib/reconstructor.cpp.orig]: no idea how it happened :| thanks for catching it up

File: src/lib/reconstructor.h.orig (hunk 2 above)
Comment (hcorg) [reply src/lib/reconstructor.h.orig]: again, little magic...

commits amended, it never happened ;)

**Review by psockoN7 — APPROVED** [review]

**Pull request merged**